test/analyticless_convergence_tests.jl

```diff
@@ -1,5 +1,5 @@
 using OrdinaryDiffEq, ParameterizedFunctions, Test, Random
-
+using ParameterizedFunctions.ModelingToolkit # macro hygiene
```

Comment:
?

Reply:
```
julia> using ParameterizedFunctions

julia> f = @ode_def LotkaVolterra begin
         dx = 1.5x - x*y
         dy = -3y + x*y
       end
ERROR: LoadError: UndefVarError: ModelingToolkit not defined
Stacktrace:
 [1] top-level scope
 [2] eval(::Module, ::Any) at ./boot.jl:331
 [3] eval_user_input(::Any, ::REPL.REPLBackend) at /home/scheme/bin/julia-1.4.0/share/julia/stdlib/v1.4/REPL/src/REPL.jl:86
 [4] run_backend(::REPL.REPLBackend) at /home/scheme/.julia/packages/Revise/BqeJF/src/Revise.jl:1184
 [5] top-level scope at none:0
in expression starting at <macrocall>:0
```

Reply:
ParameterizedFunctions should reexport MTK then. Other downstream packages might have this issue though... we should probably `@__MODULE__` in build function? That's tough